### PR TITLE
Returns and empty list of detect-manifests returns an empty string

### PR DESCRIPTION
**Pull request by klhoot**
Ensures that the list of manifest files is empty, as expected when no manifest files are found by the agent.

### CODE REVIEW

File: Corgibytes.Freshli.Cli/Services/AgentReader.cs [hunk 1]
```diff
@@ -47,7 +48,8 @@ public List<string> DetectManifests(string projectPath)
     {
         var manifests = Invoke.Command(AgentExecutablePath, $"detect-manifests {projectPath}", ".");
 
-        return manifests.TrimEnd('\n', '\r').Split("\n").ToList();
+        return manifests.IsEmpty() ? new List<string>() :
```
Comment (edwinkortman) [inline Corgibytes.Freshli.Cli/Services/AgentReader.cs:51]: Could we add a test for this?

Comment (klhoot) [reply Corgibytes.Freshli.Cli/Services/AgentReader.cs:51]: Good suggestion!

File: Corgibytes.Freshli.Cli.Test/Services/AgentReaderTest.cs [hunk 2]
```diff
@@ -44,8 +44,31 @@ public void GenerateBillOfMaterialsUsingProtobuf()
         checkoutDirectory.Delete(true);
     }
 
+    [Fact]
+    public void AgentReaderReturnsEmptyListWhenNoManifestsFound()
+    {
+        var checkoutLocation = CreateCheckoutLocation(out var checkoutDirectory);
```
Comment (edwinkortman) [inline Corgibytes.Freshli.Cli.Test/Services/AgentReaderTest.cs:50]:
```suggestion
        var checkoutLocation = CreateCheckoutLocation(out checkoutDirectory);
```

**Review by edwinkortman — APPROVED** [review]
Nice work! :D

**Pull request merged**